### PR TITLE
Add FSM comparison table

### PR DESCRIPTION
**Pull request by davidkpiano**

### CODE REVIEW

File: packages/xstate-fsm/README.md [hunk 1]
```diff
@@ -13,7 +13,29 @@
...
+| Eventless transitions       |       ❌        |                       ✅                        |
+| Nested states               |       ❌        |                       ✅                        |
+| Parallel states             |       ❌        |                       ✅                        |
+| History states              |       ❌        |                       ✅                        |
+| Final states                |       ❌        |                       ✅                        |
+| Context                     |       ✅        |                       ✅                        |
+| Entry actions               |       ✅        |                       ✅                        |
+| Exit actions                |       ✅        |                       ✅                        |
+| Transition actions          |       ✅        |                       ✅                        |
+| Parameterized actions       |       ❌        |                       ✅                        |
```
Comment (yoav-lavi) [inline packages/xstate-fsm/README.md:34]: Both for this point and Parameterized Guards - 
The official docs don't seem to reference these by this name, is this referring to [Custom Guards](https://xstate.js.org/docs/guides/guards.html#serializing-guards)?

File: packages/xstate-fsm/README.md [hunk 2]
```diff
@@ -13,7 +13,29 @@
 
 This package contains a minimal, 1kb implementation of [XState](https://github.com/davidkpiano/xstate) for **finite state machines**.
 
-**Features:**
+## Features
+
+|                             | **@xstate/fsm** | [XState](https://github.com/davidkpiano/xstate) |
```
Comment (yoav-lavi) [inline packages/xstate-fsm/README.md:18]: Suggestion - May be helpful for future readers if each of these was a link to the official docs (may be able to make a separate PR for that later if it doesn't get added to this one but is a wanted addition)

**Review by yoav-lavi — COMMENTED** [review]
Really great addition, many thanks @davidkpiano!

**Comment by yoav-lavi** [comment]
The `guards` property in the configuration also does not seem to be supported in `@xstate/fsm` but is supported in `xstate` (core)

File: packages/xstate-fsm/README.md [hunk 3]
```diff
@@ -13,7 +13,29 @@
...
+| Final states                |       ❌        |                       ✅                        |
+| Context                     |       ✅        |                       ✅                        |
+| Entry actions               |       ✅        |                       ✅                        |
+| Exit actions                |       ✅        |                       ✅                        |
+| Transition actions          |       ✅        |                       ✅                        |
+| Parameterized actions       |       ❌        |                       ✅                        |
+| Transition guards           |       ✅        |                       ✅                        |
+| Parameterized guards        |       ❌        |                       ✅                        |
+| Spawned actors              |       ❌        |                       ✅                        |
+| Invoked actors              |       ❌        |                       ✅                        |
```
Comment (tomByrer) [inline packages/xstate-fsm/README.md:39]: I think your excellent chart makes most of the list below redundant.

**Review by tomByrer — COMMENTED** [review]
I like the chart!

Should it be mentioned that "Most exports from XState are not included."?  I was disappointed when I saw `State` was not exported; I wanted to [persist state](https://xstate.js.org/docs/guides/states.html#persisting-state) in FSM.

File: packages/xstate-fsm/README.md (hunk 1 above)
Comment (Andarist) [reply packages/xstate-fsm/README.md:34]: I believe this refers to parametrized actions as in:
```
actions: {
  type: 'actionName',
  param: 42.
}
```

That being said - what you refer to (under a custom guards name) is also not supported in `@xstate/fsm` though. Currently, only `actions` can be "provided" through the second argument of the `createMachine` API.

**Review by Andarist — APPROVED** [review]

**Comment by yoav-lavi** [comment]
@davidkpiano @Andarist Hey! Reminding about this PR, do we want to make the suggested changes / merge as is?

**Comment by yoav-lavi** [comment]
@davidkpiano hey, pinging regarding this PR since it was approved

**Pull request merged**